circle.yml

```diff
@@ -326,3 +336,62 @@ jobs:
               python3 scripts/upload-to-github-release.py --file MapboxGLAndroidSDK/build/intermediates/cmake/release/debuggable-objects.tar.gz --release $CIRCLE_TAG --project mapbox-gl-native-android
             fi
       - generate-api-javadoc
+
+  #-------------------------------------------------------------------------------
+  android-appcenter-weekly:
+    parameters:
+      stl:
+        type: string
+        default: "c++_static"
+      image:
+        type: string
+        default: android-ndk-r21b:a111a5b966
+      abi:
+        type: string
+        default: "arm-v7"
+    docker:
+      - image: mbgl/<< parameters.image >>
+    resource_class: xlarge
+    working_directory: /src
+    environment:
+      LIBSYSCONFCPUS: 8
+      JOBS: 8
+      BUILDTYPE: Debug
+      IS_LOCAL_DEVELOPMENT: false
+      MBGL_ANDROID_STL: << parameters.stl >>
+    steps:
+      - install-dependencies: { gradle: true }
+      - run:
+          name: Initialize vendor submodules
+          command: git submodule update --init --recursive vendor
+      - run:
+          name: Build libmapbox-gl.so for << parameters.abi >>
+          command: make android-lib-<< parameters.abi >>
+      - run:
+          name: Build Test APK
+          command: |
+            if [ -n "${MAPBOX_DEVELOPER_CONFIG_XML}" ]; then
+              echo "${MAPBOX_DEVELOPER_CONFIG_XML}" > MapboxGLAndroidSDKTestApp/src/main/res/values/developer-config.xml
+              make android-ui-test-<< parameters.abi >>
+            fi
+      - save-dependencies: { gradle: true }
+      - run:
+          name: Run instrumentation tests on App Center
+          no_output_timeout: 20m
```

Comment:
Are we sure this is long enough to execute all devices (even is they are started parallel)?

Reply:
It will be timeout if there is no output, as shown in this [build ](https://app.circleci.com/pipelines/github/mapbox/mapbox-gl-native-android/945/workflows/329984e2-5736-49de-9657-761c486d6e2b/jobs/2443/steps) app center will continue printing waiting message, so it was not timeout even for one hour and half